### DISCUSSION TITLE
Sessions vs Traces?

### DISCUSSION THREAD

**Discussion by Manouchehri** [langfuse/langfuse#1081] [Support] [answered]
I'm having a hard time understanding if I should use a session or trace in this example.

1. Call to `gpt-4-1106-preview` writes an email.
2. Call to `gpt-3.5-turbo-1106` formats the email from the previous step.

Would it make sense to bundle these together in a session, or in a single trace?

**Comment by Manouchehri** [comment]
cc @krrishdholakia

**Comment by marcklingen** [comment, marked as answer]
If it's a single workflow / user-interaction, most users put it into a single trace.
